Vectors.Rmd

```diff
@@ -5,13 +5,13 @@ source("common.R")
 ```
 ## Introduction
 
-This chapter discusses the most important family of data types in base R: the vector types[^node]. You've probably used many (if not all) of the vectors before, but you may not have thought deeply about how they are interrelated. In this chapter, I won't cover individual vectors types in too much depth. Instead, I'll show you how they fit together as a whole. If you need more details, you can find them in R's documentation.
+This chapter discusses the most important family of data types in base R: vectors.[^node] While you've probably already used many (if not all) of the different types of vectors, you may not have thought deeply about how they're interrelated. In this chapter, I won't cover individual vectors types in too much detail, but I will show you how all the types fit together as a whole. If you need more details, you can find them in R's documentation.
```

Comment:
The footnote should go before the `.` (at least in my style)

Reply:
OK. That's not even a British-ism, I think.